lib/content-services/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -0,0 +1,220 @@
+$theme-blue-button-color: #1f74db;
+
+$dialog-large-l-width: 1075px;
+$dialog-large-l-height: 907px;
+
+$dialog-medium-l-width: 691px;
+$dialog-medium-l-height: 778px;
+
+$dialog-small-l-width: 461px;
+$dialog-small-s-width: 346px;
+$dialog-small-l-height: 432px;
+$dialog-small-s-height: 360px;
+
+$l-screen: 1920px;
+$m-screen: 1680px;
+$s-screen: 1440px;
+$xs-screen: 375px;
+
+$breakpoint-alert-with-additional-buttons-centered: 1469px;
+$breakpoint-medium-with-additional-buttons-centered: 1469px;
+
+$dialog-padding: 24px;
+
+.adf-dialog-container {
+    min-width: calc(269px - $dialog-padding * 2);
+    min-height: calc(300px - $dialog-padding * 2);
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    max-height: 100%;
+    max-height: calc(84vh - $dialog-padding * 2);
+}
+
+.adf-dialog-header,
+.adf-dialog-content,
+.adf-dialog-actions {
+    margin: 0;
+    padding: 0;
+}
+
+.adf-dialog-header {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: $dialog-padding;
+
+    &.adf-centered-header {
+        align-items: flex-start;
+    }
+
+    .adf-dialog-title {
+        font-size: large;
+        font-weight: 200;
+        margin: 0;
+    }
+
+    .adf-dialog-header-icon {
+        width: 48px;
+        height: 48px;
+        font-size: 48px;
+        color: $theme-blue-button-color;
+    }
+}
+
+.adf-description {
+    padding-top: $dialog-padding;
+}
+
+.adf-dialog-content {
+    margin: $dialog-padding 0;
+    overflow: auto;
+    flex: 1;
+    max-height: none;
+    line-height: 20px;
+}
+
+.adf-dialog-header,
+.adf-dialog-actions {
+    &::after {
+        content: ' ';
+        position: absolute;
+        left: -$dialog-padding;
+        display: block;
+        height: 0;
+        border-bottom: 1px solid #2121211f;
+        width: calc(100% + $dialog-padding * 2);
+    }
+}
+
+.adf-dialog-actions::after {
+    top: 0;
+}
+
+.adf-dialog-header::after {
+    bottom: 0;
+}
+
+.adf-dialog-actions {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    padding-top: $dialog-padding;
+    text-transform: capitalize;
+    min-height: auto;
+
+    /* stylelint-disable-next-line selector-class-pattern */
+    .mat-button.mat-button-base + .mat-button-base {
+        margin-left: 12px;
+    }
+
+    /* stylelint-disable-next-line selector-class-pattern */
+    .mat-button {
+        text-transform: capitalize;
+    }
+}
+
+/* stylelint-disable selector-class-pattern */
+.mat-dialog-container {
```

Comment:
these mat classes will be changed in ng15, can we try to avoid them?

Reply:
Aaccording to [the design](https://www.figma.com/file/ovO4EkRSqFTroCa5gYDAsx/❖-Alfresco.-Design-system-2.0?type=design&node-id=1319-324088&mode=design&t=k1dNW4GJB0O7pOaD-0) we have to override dialog `border-radius` and `margin` between buttons. 
For border-radius unfortunately I can't use `panelClass` because border-radius is used on `mat-container `element.
Do you have any suggestions how to do it?